commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/TwoSidedDrive.scala

```diff
@@ -36,7 +36,8 @@ trait TwoSidedDriveProperties extends UnicycleProperties {
 /**
   * A drivetrain with two side control (such as a tank drive)
   */
-trait TwoSidedDrive extends UnicycleDrive { self =>
+abstract class TwoSidedDrive(updatePeriod: Time)(implicit clock: Clock)
```

Comment:
I wanted to be able to take the clock as an implicit parameter directly to simplify user-side code. Also, the updatePeriod is configurable by the user, it isn't something that should be brought in from scope.